### DISCUSSION TITLE
关于世界币 - 他想干啥

### DISCUSSION THREAD

**Discussion by guttasingapore** [RoadSideUniversity/ClubHouse2024#13] [Ideas]
世界币想干啥
https://worldcoin.org/blog/announcements/new-state-bridge-update-enables-permissionless-integration-world-id
https://twitter.com/worldcoin/status/1712451900656750608

**Comment by guttasingapore** [comment]
got an invite to apply Orb, becoming a Worldcoin Operator?

**Reply by guttasingapore** [reply]
https://worldcoin.org/tech-tree#operations
https://docs.worldcoin.org/reference/api
https://docs.worldcoin.org/id/sign-in

**Reply by Lewisrxliu** [reply]
> https://worldcoin.org/tech-tree#operations
> 
> https://docs.worldcoin.org/reference/api
> 
> https://docs.worldcoin.org/id/sign-in

Further study their tech-tree...
https://whitepaper.worldcoin.org/advancing-decentralization#core-orb-engineering-source-available

**Comment by Lewisrxliu** [comment]
虹膜扫描软件开源了 - The core components of the orb's software that are now open source

Open sourcing the Orb ⚪ ,the newest episode of Worldcoin's video podcast series to learn about the orb's open source codebase ⬇️

https://youtu.be/50L_1P7yQRY

The Orb's software is now open source 🫡

https://worldcoin.org/blog/engineering/worldcoin-foundation-open-sources-core-components-orb-software
